scvi/autotune/_autotune.py

```diff
@@ -0,0 +1,289 @@
+"""Main class and helper functions."""
+import inspect
+import logging
+import os
+import warnings
+from typing import List, Optional, Union
+
+import anndata
+import numpy as np
+import ray
```

Comment:
since this is an optional dependency we should be able to capture the error and tell people to install it